src/Routes.js

```diff
@@ -158,15 +162,15 @@ export const Routes = (props) => {
                     path={paths.packageDetail.to}
                     component={PackageDetail}
                 />
-                <Route
+                {isPatchSetEnabled && <Route
                     exact
                     path={paths.patchSet.to}
                     component={PatchSet}
-                />
+                />}
 
                 <Route
                     render={() =>
-                        some(paths, p => p.to === path) || (
+                        (!isPatchSetEnabled || !some(paths, p => p.to === path)) && (
```

Comment:
I think
`condition ? whenTrue : whenFalse`
is clearer than
`!condition || whenFalse && whenTrue`